src/components/exploreBar.vue

```diff
@@ -121,44 +121,48 @@ export default {
 }
 .explore-bar-wrapper {
   position: fixed;
-  font-size: 1em;
+  font-size: 1.5em;
```

Comment:
make nav tabs display flex

Reply:
<img width="882" alt="Screen Shot 2020-02-27 at 10 24 34 PM" src="https://user-images.githubusercontent.com/44561376/75515855-f3a44780-59af-11ea-9d29-41112e677828.png">
`display: flex` looks like it causes mobile view to break.. I'm not sure why we might need..?

Reply:
it's nav-links actually but it's already display flex. i wanted to try justify content center but it looks like space evenly as it is is good enough